ansible/README.md

```diff
@@ -52,37 +66,80 @@ newgrp lxd
+First make sure that docker and docker-py are installed. Docker is used for building packages on localhost.
 
 ```
-ansible-playbook  -i hosts/lxd_hosts_from_local.txt xroad_dev.yml
+ansible-playbook  -i hosts/lxd_hosts.txt xroad_dev.yml
 ```
+
+This updates **all X-Road related packages to their latest versions** using **locally built X-Road packages** (as long as `compile-servers` group in ansible inventory = `localhost`). 
+Package version names are formed using current git commit timestamp and hash. This means that
+
+* if you only make local changes without performing a git commit, package names and version numbers are
+not changed - and executing xroad_dev playbook will keep the old packages intact 
+* if you perform a git commit (of any kind), all local modifications will be packaged and deployed 
+using the new version number - and all local modifications (whether they were included in the commit or not)
+will be deployed
+
+In short, to see changes, **a git commit must be done**.
+ 
 ### Partial compilation and deployment
 
-For fast development, you can compile and install modules separately. The ansible playbook *xroad_dev_partial.yml* offers this. For example, if you make a change to a Java or Ruby file under proxy-ui, use the following command to compile the war and deploy it to the security servers.
+For fast development, you can compile and install modules separately. The ansible playbook `xroad_dev_partial.yml` offers this. For example, if you make a change to a Java or Ruby file under proxy-ui, use the following command to compile the war and deploy it to the security servers.
 ```
-ansible-playbook  -i hosts/lxd_hosts_from_local.txt   xroad_dev_partial.yml   -e selected_modules=proxy-ui
+ansible-playbook  -i hosts/lxd_hosts.txt   xroad_dev_partial.yml   -e selected_modules=proxy-ui
 ```
 
 It is also possible to compile and install several modules (jars or wars). The following command compiles and installs common-util and signer jars and proxy-ui war to the correct servers.
 ```
-ansible-playbook  -i hosts/lxd_hosts_from_local.txt   xroad_dev_partial.yml   -e selected_modules=common-util,proxy-ui,signer
+ansible-playbook  -i hosts/lxd_hosts.txt   xroad_dev_partial.yml   -e selected_modules=common-util,proxy-ui,signer
 ```
 
-The modules are listed in dicts common_modules.yml, cs_modules.yml, cp_modules.yml and ss_modules.yml.
+This updates the **selected modules (jars or wars)** to ones compiled locally. 
+**No git commits are needed** to see changes.
+
+The modules are listed in dicts `common_modules.yml`, `cs_modules.yml`, `cp_modules.yml` and `ss_modules.yml`.
+
+Note that the playbook `xroad_dev_partial.yml` only copies jars and wars to the servers in correct locations. For the full installation, use `xroad_dev.yml`.
+
 
-Note that the playbook *xroad_dev_partial.yml* only copies jars and wars to the servers in correct locations. For the full installation, use *xroad_dev.yml*.
+### LXD Test automation environment deployment
```

Comment:
This was removed in `PVAYLADEV-926	Removed automated testing environment Ansible setup from the public repository`, right?